src/pages/atomicdex/api/v20-dev/lightning/channels/index.mdx

```diff
@@ -0,0 +1,924 @@
+| node_address      | object  | A standard `LightningOpenChannelsFilter` object.                                         |
+| amount            | object  | A standard [LightningChannelAmount](/atomicdex/api/v20#lightning-channel-amount) object. |
+| push_msat         | integer | Optional. The amount in millisatoshi to push to the counterparty while openning, to create inbound liquidity for the channel. Using the `push_msat` parameter avoids having to send funds in a separate request later.                                          |
+| channel_options   | object  | Optional. A standard [LightningChannelOptions](/atomicdex/api/v20#lightning-channel-options) object. |
+| channel_configs   | object  | Optional. A standard [LightningChannelConfig](/atomicdex/api/v20#lightning-channel-config) object.   |
+| limit             | integer | Optional, defaults to `10`. Maximum number of results to return.                         |
+
+
+#### 📌 Example using an exact amount
+
+<CodeGroup title="" tag="POST" label="METHOD_NAME">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::open_channel",
+    "params": {
+        "coin": "tBTC-lightning",
+        "node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735",
+        "amount": {
+            "type": "Exact",
+            "value": 0.004
+        }
+    },
+    "id": 22
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"uuid": "434681f8-95e5-484f-af49-fa80d8ae857b",
+		"node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@233.112.95.142:9735"
+	},
+	"id": 22
+}
+```
+</CollapsibleSection>
+
+
+#### 📌 Example with all available configuration options
+
+<CodeGroup title="" tag="POST" label="METHOD_NAME">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::open_channel",
+    "params": {
+        "coin": "tBTC-lightning",
+        "node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735",
+        "amount": {
+            "type": "Max"
+        },
+        "push_msat": 0,
+        "channel_options": {
+            "proportional_fee_in_millionths_sats": 0,
+            "base_fee_msat": 1000,
+            "cltv_expiry_delta": 72,
+            "max_dust_htlc_exposure_msat": 5000000,
+            "force_close_avoidance_max_fee_satoshis": 1000
+        },
+        "channel_configs" : {
+            "counterparty_locktime": 144,
+            "our_htlc_minimum_msat": 1,
+            "negotiate_scid_privacy": false,
+            "max_inbound_in_flight_htlc_percent": 10,
+            "announced_channel": false,
+            "commit_upfront_shutdown_pubkey": true
+        }
```

Comment:
`inbound_channels_confirmations` and `their_channel_reserve_sats` are missing in this example.

Reply:
added, thanks.